packages/smooth_app/lib/helpers/text_input_formatters_helper.dart

```diff
@@ -9,24 +9,20 @@ import 'package:smooth_app/helpers/strings_helper.dart';
 /// This separator is based on [NumberFormat], so it can be "." or "," depending
 /// on the user's language.
 ///
-/// Only languages with a space character (eg: French) as a group separator will
-/// have access to all features.
+/// It works if there are no group separator in the format, as it would be
+/// problematic with the `123,456.9` and both comma and dot.
 ///
-/// Also, if a separator is already displayed, it will be move to the new
+/// Also, if a separator is already displayed, it will be moved to the new
 /// position
 class DecimalSeparatorRewriter extends TextInputFormatter {
```

Comment:
It would be great to have unit tests for those functions, as it would be easy to break something that works today in future changes.